### PR TITLE
chore: add alignment to toolbar button

### LINKED ISSUE #32309: [Feat]: Toolbar Buttons widget horizontal alignment
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Allow users to align button in Toolbar widget left or right.

E.g. in case on the screenshot I would like to be able to align all buttons to the right.

image

We can reuse the 'position' control (screenshot from Statbox's property pane):

### Public Sample App

_No response_

### Environment

Deploy Preview

### Severity

Medium (Frustrating UX)

### Issue video log

_No response_

### Version

cloud

### PR DESCRIPTION
**Pull request by jsartisan**
Fixes #32309

/ok-to-test tags="@tag.Anvil"
> [!TIP]
> 🟢 🟢 🟢 All cypress tests have passed! 🎉 🎉 🎉
> Workflow run: <https://github.com/appsmithorg/appsmith/actions/runs/9127236127>
> Commit: 016f9d08fbdd87d6ff355d45e239f015d9401612
> Cypress dashboard url: <a href="https://internal.appsmith.com/app/cypress-dashboard/rundetails-65890b3c81d7400d08fa9ee5?branch=master&workflowId=9127236127&attempt=1" target="_blank">Click here!</a>

### CODE REVIEW

File: app/client/packages/design-system/widgets/src/components/ActionGroup/src/types.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import type { ButtonGroupProps } from "../../ButtonGroup";
+
+export const ACTION_GROUP_ALIGNMENTS = {
+  start: "Start",
+  center: "Center",
```
Comment (KelvinOm) [inline app/client/packages/design-system/widgets/src/components/ActionGroup/src/types.ts:5]: Why do we need this option? I don't see us using this in the app.

File: app/client/src/widgets/wds/WDSToolbarButtonsWidget/widget/types.ts [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
 
 export type ButtonsList = Record<string, ToolbarButtonsItemComponentProps>;
 
 export interface ButtonGroupWidgetProps extends WidgetProps {
-  buttonColor: ButtonGroupProps<object>["color"];
-  buttonVariant: ButtonGroupProps<object>["variant"];
-  orientation: ButtonGroupProps<object>["orientation"];
+  buttonColor: ActionGroupProps<object>["color"];
+  buttonVariant: ActionGroupProps<object>["variant"];
+  orientation: ActionGroupProps<object>["orientation"];
```
Comment (KelvinOm) [inline app/client/src/widgets/wds/WDSToolbarButtonsWidget/widget/types.ts:10]: Let's remove orientation setting for Toolbar buttons. Now alignment doesn't work with vertical orientation. I discussed it with Taras. 

https://github.com/appsmithorg/appsmith/assets/11555074/aa2d75d2-b201-4eba-96d8-7fb082083b7a

**Review by KelvinOm — CHANGES_REQUESTED** [review]

File: app/client/src/widgets/wds/WDSToolbarButtonsWidget/widget/types.ts (hunk 2 above)
Comment (jsartisan) [reply app/client/src/widgets/wds/WDSToolbarButtonsWidget/widget/types.ts:10]: Removing it. Btw, another option could be hiding the alignment control in property pane when the orientation is vertical.

Comment (KelvinOm) [reply app/client/src/widgets/wds/WDSToolbarButtonsWidget/widget/types.ts:10]: We discussed hiding option, we don't need it.

**Review by KelvinOm — APPROVED** [review]

**Pull request merged**